azure_arc_symphony_jumpstart/Bootstrap.ps1

```diff
@@ -0,0 +1,104 @@
+    Remove-Item .\AzureCLI.msi
+ 
+    # Apply PATH to current session right away as the auto-updated system PATH won't take effect until next session
+    if (-not $env:Path.Contains("C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin")) {
+        $env:Path="$env:Path;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin"
+    }
+}
+else
+{
+    Write-Host "Azure CLI is already installed"
+}
+
+az --version
+# Login as service principal
+az login --service-principal --username $spnClientId --password=$spnClientSecret --tenant $spnTenantId
+
+az provider register --namespace Microsoft.Kubernetes
+az provider register --namespace Microsoft.KubernetesConfiguration
+az provider register --namespace Microsoft.ExtendedLocation
+az provider register --namespace Microsoft.ToolchainOrchestrator
+
+az provider show -n Microsoft.Kubernetes -o table
+az provider show -n Microsoft.KubernetesConfiguration -o table
+az provider show -n Microsoft.ExtendedLocation -o table
+az provider show -n Microsoft.ToolchainOrchestrator -o table
+
+# Installing Azure CLI extensions
+Write-Host "`n"
+az extension add --name "connectedk8s" -y
+az extension add --name "k8s-extension" -y
+az extension add --name "customlocation" -y
+
+# Start to arc enable the cluster and install extension
+az account set -s $subscriptionId
+az aks get-credentials --name $clusterName --resource-group $resourceGroup --overwrite-existing --admin
+# Add cert to deal with connection issue in VM
+Invoke-WebRequest -Uri https://secure.globalsign.net/cacert/Root-R1.crt -OutFile "$tempDir\globalsignR1.crt"
+Import-Certificate -FilePath "$tempDir\globalsignR1.crt" -CertStoreLocation Cert:\LocalMachine\Root 
+az connectedk8s connect -g $resourceGroup -n $clusterName --location $azureLocation
+az k8s-extension create `
+    --resource-group $resourceGroup `
+    --cluster-name $clusterName `
+    --cluster-type connectedClusters `
+    --name $extensionName `
+    --extension-type Microsoft.ToolchainOrchestrator `
+    --scope cluster `
+    --release-train dev `
+    --version $extensionVersion `
+    --auto-upgrade false
+
+az k8s-extension show --resource-group $resourceGroup --cluster-name $clusterName --cluster-type connectedClusters --name $extensionName
+
+az connectedk8s enable-features -n $clusterName -g $resourceGroup --features cluster-connect custom-locations
+### By adding --namespace you can bound the namespace to the custom location being created, by default it will use your custom location name.###
+
+$KUBECONFIG = "C:\Users\$env:USERNAME\.kube\config"
+$hostResourceId = "/subscriptions/$subscriptionId/resourceGroups/$resourceGroup/providers/Microsoft.Kubernetes/connectedClusters/$clusterName"
+$clusterExtensionId = "/subscriptions/$subscriptionId/resourceGroups/$resourceGroup/providers/Microsoft.Kubernetes/connectedClusters/$clusterName/Providers/Microsoft.KubernetesConfiguration/extensions/$extensionName"
+az customlocation create `
```

Comment:
Where do you create toolchain orchestrator resources? Is this not the part of jumpstart?